modules/lang-groovy/src/test/java/org/elasticsearch/script/groovy/GroovyIndexedScriptTests.java

```diff
@@ -136,7 +136,7 @@ public void testDisabledUpdateIndexedScriptsOnly() {
         client().prepareIndex("test", "scriptTest", "1").setSource("{\"theField\":\"foo\"}").get();
         try {
             client().prepareUpdate("test", "scriptTest", "1")
-                    .setScript(new Script("script1", ScriptService.ScriptType.STORED, GroovyScriptEngineService.NAME, null)).get();
+                    .setScript(new Script("script1", ScriptService.ScriptType.STORED, "groovy", null)).get();
```

Comment:
I think you should still be able to use the constant here? In case one day we decide to rename it "GrOoVy"